src/calculateRentalCost.js

```diff
@@ -28,7 +28,13 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days > 0 && days < 3) {
+    return 40 * days;
+  } else if (days >= 3 && days < 7) {
+    return 40 * days - 20;
+  } else {
+    return 40 * days - 50;
+  }
```

Comment:
create variables for all numbers
create a variable for `40 * days`, you repeat this three times